daemon/src/peer/run_state/config.rs

```diff
@@ -0,0 +1,90 @@
+//! Configuration types for configuring the `RunState`.
+
+use std::time::Duration;
+
+/// Default time to wait between announcement subroutine runs.
+pub(super) const DEFAULT_ANNOUNCE_INTERVAL: Duration = std::time::Duration::from_secs(1);
```

Comment:
Are these used outside of this file?

Reply:
bump

Reply:
I don't think so. I'll privatise the sector